lib/job.js

```diff
@@ -246,11 +246,15 @@ class Job extends BaseModel {
 
                     return Promise.all(
                         builds.map(build => {
-                            this[executor].stop({
-                                buildId: build.id,
-                                pipelineId: pipeline.id,
-                                jobId: this.id
-                            });
+                            const stoppedStatuses = ['ABORTED', 'FAILURE', 'SUCCESS'];
```

Comment:
Do we need to add `UNSTABLE` and `COLLAPSED` ? These are also kind of finished status.

Reply:
`UNSTABLE` seems not to be only finished status.
https://github.com/screwdriver-cd/ui/blob/0111bc864056fca3d53c5461e9576e65d45be7f3/app/utils/build.js#L11

Reply:
we should skip unstable, since buld status can be set as unstable in the middle of a build and still have rest of steps running.

Reply:
collapsed should be added though

Reply:
I added `COLLAPSED` status.